<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="coinsInput">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter the amount of your coins :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="convertButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
   <widget class="QComboBox" name="coinsBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Dollar</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SyrianPound</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Euro</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SaudiRial</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>KuwaitDinar</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>UAEDirham</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>JordanDinar</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert to ...</string>
    </property>
   </widget>
   <widget class="QLabel" name="couinsValueLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Value :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>
Convert your currency</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert your currency</string>
    </property>
   </widget>
   <widget class="QComboBox" name="coinsBox_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Dollar</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Euro</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SaudiRial</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>KuwaitDinar</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>UAEDirham</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>JordanDinar</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SyrianPound</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>494</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
